cicd/gitops/configuring-resource-quota.adoc

```diff
@@ -0,0 +1,14 @@
+:_content-type: ASSEMBLY
+[id="configuring-resource-quota"]
+= Configuring resource quota or requests
+include::_attributes/common-attributes.adoc[]
+:context: configuring-resource-quota
+
+toc::[]
+
+[role="_abstract"]
+The Argo CD Custom Resource allows you to create, update and delete resource requests and limits for Argo CD workloads. 
```

Comment:
The IBM SG recommends against using "allows" in this context - so maybe:

With the Argo CD Custom Resource, you can  create, update and delete resource requests and limits ...."

allow verb
Avoid stating that inanimate objects grant abilities to people, as in "the product allows you to...". Whenever possible, use a direct, user-focused alternative such as "you can use the product to..." or "with this product, users can..."